.travis.yml

```diff
@@ -24,6 +24,7 @@ env:
     - TASK=mpi METHOD=pip
     - TASK=gpu METHOD=source PYTHON_VERSION=3.5
     - TASK=gpu METHOD=pip PYTHON_VERSION=3.6
+    - TASK=r-package R_MAC_VERSION=3.6.1 R_TRAVIS_LINUX_VERSION=3.6.1-3bionic
```

Comment:
The same about `R_MAC_VERSION` and `R_TRAVIS_LINUX_VERSION` as in https://github.com/microsoft/LightGBM/pull/2530#discussion_r353193184.

Place to define them for Travis:
https://github.com/microsoft/LightGBM/blob/69c1c33093cb9fa1afd896b8a458e47aa2930c92/.travis.yml#L39-L58

Reply:
ah I see, sure no problem I can do that